psd-web/spec/features/add_business_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "Adding a business", :with_stubbed_mailer, :with_stubbed_elasticsearch, :with_stubbed_keycloak_config, type: :feature do
+  let(:city)          { Faker::Space.planet }
+  let(:trading_name)  { Faker::Company.name }
+  let(:investigation) { create(:enquiry) }
+
+  before { sign_in }
+
+  it "allows the relevent params to be posted" do
```

Comment:
Please change: within https://github.com/UKGovernmentBEIS/beis-opss-psd/pull/329 we settled on using `scenario` within feature specs rather than `it` blocks:

```suggestion
  scenario "Adding a business" do
```